api/user/index.ts

```diff
@@ -0,0 +1,57 @@
+import * as express from 'express'
+import * as log4jui from '../lib/log4jui'
+import authInterceptor from '../lib/middleware/auth'
+const logger = log4jui.getLogger('auth')
+
+export const router = express.Router({ mergeParams: true })
+
+router.use(authInterceptor)
+
+router.get('/details', handleUserRoute)
+
+function handleUserRoute(req, res) {
+
+  const idleTimeOuts: {caseworker: number; solicitors: number; special: number} = {
```

Comment:
Move this function to util and add some tests

Reply:
this is just a constant nothing to test here - this could go to config... but have left then out when have better understanding where the constants should live... i think we need to wait the .evn cifiguration to do this